server/src/routes/api/state.mjs

```diff
@@ -39,7 +39,7 @@ function getState(req, res) {
   });
 }
 
-// POST /api/v1/global/latency
+// POST /api/v1/state/global/latency
```

Comment:
Do you have to do anything to tell your server about this new piece of the route's path? How does it know that the handlers for `api/state/` are located here? Is that just by convention of the dir structure?

Reply:
There's a "configureAPI.mjs" file that has the actual Express routes. Here, I'm just updating the comments to match the actual routes.